selftests/vmimage/tests/plugin/unit_plugin_vmimage.py

```diff
@@ -6,8 +7,16 @@
 from avocado.core.settings import Settings
 from avocado.plugins import vmimage as vmimage_plugin
 from avocado.utils import vmimage as vmimage_util
-from selftests.functional.plugin.vmimage import create_metadata_file, missing_binary
-from selftests.utils import skipOnLevelsInferiorThan, temp_dir_prefix
+from selftests.utils import missing_binary, skipOnLevelsInferiorThan, temp_dir_prefix
+
+
+def create_metadata_file(image_file, metadata):
```

Comment:
IIUIC this is a duplication from [here](https://github.com/avocado-framework/avocado/pull/6112/commits/35d3e1d4c78f014ba3d58359dcb53cf8321154fa#diff-8dad6ccb3563547560d89e94725ce7e6efd6ce70cdad21f774edce66d238c543L18), and we could use import to avoid the duplication.